### ISSUE TITLE
rabbitmq plugin: make deprecated heathcheck optional

### ISSUE THREAD

**Issue by pabigot** [influxdata/telegraf#7823]
The latest RabbitMQ servers issue a diagnostic each time the HTTP management API is used to invoke a node health check from the rabbitmq telegraf plugin.  This makes the rabbitmq server logs very noisy.

### Relevant telegraf.conf:

```toml
# Reads metrics from RabbitMQ servers via the Management Plugin
[[inputs.rabbitmq]]
  # No non-default configuration
```

### System info:

Telegraf 1.14.5 monitoring rabbitmq 3.8.5 using the rabbitmq:3-management docker image

### Steps to reproduce:

Enable `inputs.rabbitmq` with RabbitMQ 3.8.5 and monitor the logs

****
### Expected behavior:

No complaints in the rabbitmq logs due to use of the rabbitmq monitoring plugin.

### Actual behavior:

Messages like:
> 2020-07-11 15:56:20.023 [warning] <0.26026.0> rabbitmqctl node_health_check and its HTTP API counterpart are DEPRECATED. See https://www.rabbitmq.com/monitoring.html#health-checks for replacement options.

appear at the telegraf check rate of 0.1 Hz.

### Additional info:

See: rabbitmq/rabbitmq-cli#426

**Comment by ssoroka** [comment]
If it's deprecated, maybe we should just stop collecting this?

**Comment by pabigot** [comment]
> If it's deprecated, maybe we should just stop collecting this?

I think #7920 handles this: manual run with `telegraf --test` removes the diagnostic on my system.

**Comment by ssoroka** [comment]
Looks like this is already resolved as of #7920
